src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseassignment/AssignCaseToUserHandlerTest.java

```diff
@@ -118,16 +110,8 @@ void setup() {
             params = callbackParamsOf(dataMap, ASSIGN_CASE_TO_APPLICANT_SOLICITOR1_SPEC.name(), CallbackType.SUBMITTED);
         }
 
-        @Test
-        void shouldNotUpdateSpecSupplementaryDataOnSubmittedWhenGlobalSearchOff() {
-            when(toggleService.isGlobalSearchEnabled()).thenReturn(false);
-            assignCaseToUserHandler.handle(params);
-            verify(coreCaseDataService, never()).setSupplementaryData(any(), any());
-        }
-
         @Test
         void shouldReturnSpecSupplementaryDataWhenGlobalSearchEnabled() {
```

Comment:
Please rename this test by removing GlobalSearch in it's title as it is not relevant anymore
ie shouldReturnSpecSupplementaryData

Reply:
Test name updated